### DISCUSSION TITLE
Clean up changelog of a certain user

### DISCUSSION THREAD

**Discussion by MalfuncEddie** [netbox-community/netbox#5715] [Q&A] [answered]
Hi,

Due to a scripting issue a certain user has update tenant object every minute for a couple of weeks. The result is millions of lines in the changelog. When I filter on the user I get a postgres error that the tmp is to small. 

What would be the correct way to delete these entries of this user? (do a delete where in the DB ?)

kind regards,

**Comment by a31amit** [comment]
I was similar similar thing in dev database and I don’t see any problem, if you are better with managing database with sql, you can cleanup extras_object changelog table

**Comment by ryanmerolle** [comment, marked as answer]
You can do this easily at least in 3.0 since changelog section allows you to filter by user and mass delete.
